File: java/oop/StudentApp/src/com/techlabs/student/Student.java
package com.techlabs.student;

public class Student {
	private int rolNo;
	private int standard;
	private String name;
	
	public Student(String name,int rollNo,int standard){
		this.name=name;
		this.rolNo=rollNo;
		this.standard=standard;
	}
	
	public int getRollNo(){
		return this.rolNo;
	}
	public int getStandard(){
		return this.standard;
	}
	public String getName(){
		return this.name;
	}
	

}

File: dotNet/csharp/StudentApp/StudentApp/Student.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace StudentApp
{
    class Student
    {
        private int _rollNo;
        private int _standard;
        private string _name;

        public Student(string name,int rollno,int standard)
        {
            this._name = name;
            this._rollNo = rollno;
            this._standard = standard;
        }

        public int RollNo
        {
            get
            {
                return _rollNo;
            }
        }

        public int Standard
        {
            get
            {
                return _standard;
            }
        }

        public string Name
        {
            get
            {
                return _name;
            }
        }
       

    }
}
